### PR TITLE
Add real credentials for Zendesk access

### PR DESCRIPTION
**Pull request by issyl0**
- We want to start sending real feedback to Zendesk now that we're near
  go-live. This sets all the required envvars to actual values (one from
  Parameter Store).

https://trello.com/c/38BqCJIU/222-get-zendesk-working

### CODE REVIEW

File: terraform/modules/hub/files/tasks/frontend.json [hunk 1]
```diff
@@ -109,13 +112,10 @@
...
       "Name": "ZENDESK_URL",
-      "Value": "https://example.com"
+      "Value": "https://govuk.zendesk.com"
     }, {
       "Name": "ZENDESK_USERNAME",
-      "Value": "fake@zendesk.user"
-    }, {
-      "Name": "ZENDESK_TOKEN",
-      "Value": "123"
+      "Value": "ida-no-reply@digital.cabinet-office.gov.uk"
```
Comment (tlwr) [inline terraform/modules/hub/files/tasks/frontend.json]:
```suggestion
      "Value": "${var.zendesk_username}"
```

File: terraform/modules/hub/files/tasks/frontend.json [hunk 2]
```diff
@@ -109,13 +112,10 @@
       "Value": "1"
     }, {
       "Name": "ZENDESK_URL",
-      "Value": "https://example.com"
+      "Value": "https://govuk.zendesk.com"
```
Comment (tlwr) [inline terraform/modules/hub/files/tasks/frontend.json]:
```suggestion
      "Value": "https://${var.zendesk_domain}"
```

**Comment by issyl0** [comment]
Incorporated @tlwr's changes (related: https://github.com/alphagov/verify-infrastructure-config/pull/56) and we know this works because we tested sending some feedback.

**Comment by issyl0** [comment]
🙏

**Pull request merged**